src/index.html

```diff
@@ -9,6 +9,100 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class ="header">
+      <a href="#" class ="logo_link">
+        <img
+          src="/images/logo.png"
+          class="logo"
+          alt="logo"
+         >
+      </a>
+      <nav class="link_box">
+        <ul class="link_list">
+          <li
+            class="list_item"
+          >
```

Comment:
It's overkill, you can leave attribute on one line there were less then 3

Reply:
done